library/src/scripts/banner/bannerStyles.ts

```diff
@@ -254,7 +255,7 @@ export const bannerVariables = useThemeCache((forcedVars?: IThemeVariables, altN
             top: 18,
             bottom: 8,
         },
-        text: "How can we help you?",
+        text: getMeta("ui.siteName", t("How can we help you?")),
```

Comment:
So how's this going to work for multilingual? Has this been discussed?

Reply:
Multi lingual will be handled in a different sprint.